BuildResidentialHPXML/measure.rb

```diff
@@ -1091,6 +1091,12 @@ def arguments(model)
     arg.setDefaultValue(24)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument("mech_vent_used_for_seasonal_cooling_load_reduction", true)
```

Comment:
This shouldn't be a measure argument.

For mechanical ventilation, you'll always want to set `used_for_whole_building_ventilation` to True. For whole house fans, if you decide to allow them in the measure, you'll always want to set `mech_vent_used_for_seasonal_cooling_load_reduction` to true.

Reply:
I'm not totally sure what you mean. So you're saying we should have a separate argument where you can set either "mechanical ventilation" or "whole house fan"?

Reply:
You are already calling out a bunch of inputs as "Mechanical Ventilation: Foo". For those inputs, they should create a HPXML ventilation fan where `used_for_whole_building_ventilation` is true. Down the road, if you add "Whole House Fan: Foo" inputs, then you'd create a separate HPXML ventilation fan where `used_for_seasonal_cooling_load_reduction` is true.

Reply:
Oh. So a "whole house fan" is not considered "mechanical ventilation" even though it falls into the MechanicalVentilation block of HPXML?

Reply:
I agree that HPXML is a little confusing in this regard. But yes, you could end up with two VentilationFan objects in HPXML's MechanicalVentilation section, one for what we call mechanical ventilation and one for a whole house fan.

Reply:
Should we have a test file that includes both?

Reply:
I don't think it's necessary.